ExtraDry/ExtraDry.Core.Tests/Internals/LinqBuilderTests.cs

```diff
@@ -1,6 +1,9 @@
 ﻿using ExtraDry.Server.Internal;
+using System;
```

Comment:
No necessary for this PR, but as we've moved to .NET 6 I'm also starting to move to implicit using, global usings (for common namespaces), and file-level namespace declarations.  Cuts down on the fluff that C# inherited from C++.